app/models/katello/host/subscription_facet.rb

```diff
@@ -78,9 +79,9 @@ def update_facts(rhsm_facts)
       def self.find_host(name, organization)
         hosts = ::Host.where(:name => name)
         return nil if hosts.empty? #no host exists
-        if hosts.where(:organization_id => organization.id).empty? #not in the correct org
-          #TODO http://projects.theforeman.org/issues/11532
-          fail "Can't handle registering to host in a different org, need to handle this case."
+        if hosts.where("organization_id = #{organization.id} OR organization_id is NULL").empty? #not in the correct org
+          #TODO: http://projects.theforeman.org/issues/11532
+          fail _("Host is currently registered to a different org, please migrate host to %s.", organization.name)
```

Comment:
I was just inventing a new interpolation method.  Its not my fault ruby hasn't implemented it yet